handlers/challenge_handler.py

```diff
@@ -72,6 +76,10 @@ def execute(self, slack_wrapper, args, channel_id, user_id):
             raise InvalidCommand(
                 "Command failed. Challenge name must be <= 10 characters.")
 
+        # Check for invalid characters
+        if re.match('^[\w\-_]+$', new_name) is None:
```

Comment:
Should this either be a helper function, or at the very least put the regex into a variable since we repeat it a lot.